### DISCUSSION TITLE
Realtek network adapter can't be recognized

### DISCUSSION THREAD

**Discussion by gary21828** [SpiralLinux/SpiralLinux-project#188] [General]
It may be a Debian bug. I tried to install Debian non-free net install CD and the latest Spiral Linux into a Dell Optiplex 3020, which had a Realtek network adapter 8168 or similar. The systems can't recognize the network adapter. But the last version of Spiral Linux and Debian non-free live version work. I downloaded the Realtek non-free firmware deb (2019) and put it on Debian non-free net install CD. The problem was solved.

**Comment by geckolinux** [comment]
Hi there, it looks like this upstream bug:

- https://github.com/SpiralLinux/SpiralLinux-project/issues/175
- https://github.com/orgs/SpiralLinux/discussions/176
